### ISSUE TITLE
Feedback on mini_ex4 by Rolf Holm

### ISSUE THREAD

**Issue by OorthianEmissary** [GustneGustav/Rasmus-er-sej-og-flot#8]
The program shows a geometric shape that changes between circle and square, when it hits full width or zero width. The screen changes colour at the same points as well as height zero and full height. The sliders in the program don't do anything, which tricked me into spending the better part of 20 seconds setting them to higher and lower values. I guess they were supposed to control the speed of the ball as well as other parameters of the program, but didn't get integrated in time for it to be handed in.
On the notion of feedback, it doesn't have much more feedback than the program is running, though the sliders are deceptive, as they trick the user into believe at first glance that they can interact with the program.

In conclusion; Simple and random.
